src/nvim/os_unix.c

```diff
@@ -681,6 +681,6 @@ static int have_dollars(int num, char_u **file)
 
   for (i = 0; i < num; i++)
     if (vim_strchr(file[i], '$') != NULL)
-      return TRUE;
-  return FALSE;
+      return true;
```

Comment:
stupid github won't let me put a comment there, but looks like `have_wildcard` could be converted as well.